index.html

```diff
@@ -50,6 +58,9 @@
             }
         );
     </script>
+
+    <!--a button to Google-->
+    <a href="http://www.google.com" id="a_btn">GOOGLE</a>
```

Comment:
This should have absolute position, but I will fix this in my pull request since I have to edit this anyway to include my button with the same style.